custom_addons/module_custom_k/__manifest__.py

```
@@ -0,0 +1,34 @@
+
+    # always loaded
+    'data': [
+        # 'security/ir.model.access.csv',
+        'views/views.xml',
+        'views/templates.xml',
+    ],
+    # only loaded in demonstration mode
+    'demo': [
+        'demo/demo.xml',
+    ],
+}
```

Ruff (0.11.9)

Found useless expression. Either assign it to a variable or remove it.

(B018)